modules/plugins/languages/markdown.nix

```diff
@@ -96,7 +83,7 @@ in {
       type = mkOption {
         type = enum (attrNames formats);
         default = defaultFormat;
-        description = "Markdown formatter to use";
+        description = "Markdown formatter to use. `denofmt` is deprecated and currently aliased to deno_fmt.";
```

Comment:
What's the purpose of this? If it's a warning, we should warn the user directly not to set it. Though `deno fmt` is supported for markdown, so I'm confused why we're warning the user at all?